src/components/Header/Header.stories.tsx

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+
+import { ActionsMenu } from "./ActionsMenu";
+import { Button } from "components/Button/Button";
+import { Header } from "./Header";
+
+const navData = [
+  {
+    buttonText: "Current Section",
+    current: true,
+    columns: [
+      [
+        { text: "Navigational Link", href: "" },
+        { text: "Navigational Link", href: "" },
+        { text: "Navigational Link", href: "" },
+        { text: "Navigational Link", href: "" },
+      ],
+      [{ text: "Navigational Link", href: "" }],
+      [
+        { text: "Navigational Link", href: "" },
+        { text: "Navigational Link", href: "" },
+        { text: "Navigational Link", href: "" },
+      ],
+      [
+        { text: "Navigational Link", href: "" },
+        { text: "Navigational Link", href: "" },
+      ],
+    ],
+  },
+  {
+    buttonText: "Section",
+    columns: [
+      [
+        { text: "Navigational Link", href: "" },
+        { text: "Navigational Link", href: "" },
+        { text: "Navigational Link", href: "" },
+        { text: "Navigational Link", href: "" },
+        { text: "Navigational Link", href: "" },
+        { text: "Navigational Link", href: "" },
+        { text: "Navigational Link", href: "" },
+      ],
+    ],
+  },
+];
+
+export default {
+  title: "USWDS/Base/Header",
+  component: Header,
+  args: {
+    logoProps: {
+      altText: "CMS.gov Project",
+      // source: "../../assets/img/logos/cms_logo.png",
```

Comment:
Might need cleanup? I see what you're trying to show though, the alt text would show in a typography component.